sdcm/k8s_configs/eks/scylla-node-prepare.yaml

```diff
@@ -174,8 +174,9 @@ spec:
         app: node-pkg-installer
     spec:
       containers:
-      # NOTE: image is built on Feb 1, 2023 -> scylla-operator:1.9.0-alpha.1
-      - image: "scylladb/scylla-operator@sha256:10e3e7dddc2de1bbd473710223a4be5e8c5ef5dc9bb1a61d81fdc398d9dcd74c"
+      # TODO: set the following image to some static tag that supports also ARM arch (when appears)
+      #       As of Dec 15, 2023 only the 'latest' tag does support ARM.
+      - image: "scylladb/scylla-operator:latest"
```

Comment:
Maybe we should set some specific sha of the manifest ?

Reply:
Any `sha` is single-arch.